.aspell.en.pws

```diff
@@ -59,3 +59,4 @@ CPUs
 MiB
 RBAC
 YAML
+Loki
```

Comment:
new line please

Reply:
you also need to update the total count at the top

Reply:
done